modules/ROOT/pages/import/file-provision.adoc

```diff
@@ -2,23 +2,29 @@
 = File provision
 :description: This section describes how to provide files for import.
 
+You start by connecting your data source, which can be a SQL database or local flat files.
```

Comment:
Maybe call it a relational database rather than a SQL database.

Reply:
Yep!